include/SSVOpenHexagon/Core/MenuGame.hpp

```diff
@@ -137,25 +137,97 @@ class MenuGame
     void okAction();
     void eraseAction();
     void exitAction();
-    void createProfileAction();
-    void selectProfileAction();
-    void openOptionsAction();
-    void selectPackAction();
+    int packChange{0};
```

Comment:
Could you add a comment here or use a more descriptive name? No idea what `packChange` does just by looking at the data member.

Reply:
I set up two binds to change the current pack.
One to go to the previous pack and one to go to the next.

Instead of creating two separate functions the actions "activate"
changePackAction(1) to go to the next
changePackAction(-1) to go to the previous one.

The reason for the need for the value packChange is the fact that when user asks to have the current pack changed what it actually does is triggering the animation that causes the current level list to fold and the next one to unfold.

packChange allows MenuGame to know to which pack it needs to switch once the folding animation has ended and properly execute changePack()
```
void MenuGame::changePack()
{
    const auto& p(assets.getPackInfos());
    packIdx = ssvu::getMod(packIdx + packChange, 0, p.size());
    levelDataIds = assets.getLevelIdsByPack(p.at(packIdx).id);
    setIndex(0);
}```